Makefile

```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Werror -I./inc -I./src -o main
```

Comment:
You could add more flags. Like -Wall -Wextra -pedantic -Wconversion

Reply:
Yes, when you apply @Pawellos comments, you will see that this code is not correct. I mean something in `print` function in main.

Reply:
Ok. I change Makefile and functions `print`